packages/ui-components/style/base.css

```diff
@@ -51,16 +51,6 @@ strong {
   font-weight: unset;
 }
 
-/* Override Blueprint's _typography.scss styles */
-a {
-  text-decoration: unset;
```

Comment:
Did you do a quick search to check there are no `a`-tags in core that are use for buttons and so that probably should not be decorated?

Styling rules, it seems the default behavior of browser is to break the underline when needed (as seen in your screenshot). This is recommended [there](https://www.a11y-101.com/development/underlined-links) too. But should we [enforce that rule](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-skip-ink) or let the web browser behaves as it wants?

Reply:
I modified the styling to leave most `a` tags in the application styled as they have been. I would prefer to leave the underlining behavior up to the browser.

Reply:
FYI I realized that for example firefox has a setting to toggle underlining links. So indeed best to let user do what they like through the browser.